ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron.hs

```diff
@@ -806,32 +854,98 @@ toCBORAHeaderOrBoundary pm epochSlots abob =
   Mempool integration
 -------------------------------------------------------------------------------}
 
+-- | The hash of a delegation certificate.
+type DlgCertId = Crypto.AbstractHash Blake2b_256 CC.Delegation.Certificate
+
+-- | The hash of an update vote.
+type UpdateVoteId = Crypto.AbstractHash Blake2b_256 CC.Update.Vote
+
+-- | An error type which represents either a UTxO, delegation, update proposal
+-- registration, or update vote error in the Byron era.
+data ByronApplyTxError
```

Comment:
Perhaps this should be a data type defined in `cardano-ledger` (as well as `DlgCertId` and `UpdateVoteId`).

Reply:
What are the pros/cons?

Reply:
meh, will leave `ByronApplyTxError` as is. I've already moved `DlgCertId` and `UpdateVoteId` to `cardano-ledger`.